README.md

```diff
@@ -18,9 +18,16 @@ Open this file `C:\Windows\System32\drivers\etc\hosts` in **Administrator** mode
 127.0.0.1       inner-circle.local.tourmalinecore.internal
 ```
 
+### For Windows
+
+Open this file `C:\Windows\System32\drivers\etc\hosts` in **Administrator** mode in e.g. `Notepad` and add the line.
+
```

Comment:
There was just a header and a section about how to act on Windows, why is there another one?